docs/installation/config/admin_configuration.rst

```diff
@@ -0,0 +1,181 @@
+.. _installation_admin_config:
+
+
+================================
+Open Klant configuration (admin)
+================================
+
+Before you can work with Open Klant after installation, a few settings need to be
+configured first.
+
+.. note::
+
+    This document describes the manual configuration via the admin. You can perform
+    most of this configuration via the :ref:`command line <installation_configuration_cli>`.
+    which is both faster and less error prone.
+
+.. _installation_configuration_token_authorization:
+
+Create an API Token
+===================
+
+Open Klant
+----------
+By creating an API token, we can perform an API test call to verify the successful
+installation.
+
+1. Configure the Token authorizations for the Open Klant:
+
+   a. **Api Auth** > **Token authorizations**
+   b. Select Click **Token authorization toevoegen** (or select the token if it already exists).
+   c. Fill out the form:
+
+      - **Identifier**: *For example:* ``token-1``
+      - **Contact person**: *For example:* ``Person 1``
+      - **Email**: *For example:* ``test@example.com``
+      - **Organization**: *For example:* ``Organization XYZ``
+      - **Application**: *For example:* ``Application XYZ``
+      - **Administration**: *For example:* ``Administration XYZ``
+   
+   d. Click **Opslaan**.
+
+After creating the **Token**, the **key** can be used in the ``Authorization`` header for make the requests. 
+Example of an API call here :ref:`command line <installation_configuration_api_call>`,
+
+.. _installation_configuration_sites:
+
+Setting the Domain
+==================
+
+To properly configure your application's **domain**, you should use environment variables 
+instead of the deprecated `django.contrib.sites` library to manage domain configurations.
+
+To see the list of available environment variables and how they are specified, click :ref:`here <installation_env_config>`.
```

Comment:
Because the `SITE_DOMAIN` is only needed for `register_kanalen`, we can put this section under `Configure Notificaties API`, because it's only needed if notifications are enabled.

Also I don't think we have to mention `django.contrib.sites`, we can just say that `SITE_DOMAIN` needs to be configured and link to the envvar docs.

Let's also use double backticks around these kind of values (``), because single backticks are rendered as italic in RST (different from markdown):

![afbeelding](https://github.com/user-attachments/assets/107ef8be-16b3-42d9-9c73-5938241b3456)